### PR TITLE
Add kustomize to deploy examples

### PR DESCRIPTION
**Pull request by aneeshkp**
Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -32,7 +32,20 @@ Use producer.yaml,consumer.yaml and service.yaml from examples/manifests folder
...
+$ curl -s "https://raw.githubusercontent.com/\
+kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+ 
+$mv kustomize /usr/loca/bin/
 
+```
+$make deploy-examples
+
+### Delete deployment 
+$make undeploy-examples
```
Comment (jzding) [inline docs/development.md]: same here 

> $make undeploy-example

File: docs/development.md [hunk 2]
```diff
@@ -32,7 +32,33 @@ Use producer.yaml,consumer.yaml and service.yaml from examples/manifests folder
...
 
 
+### Deploying examples using kustomize
 
+Install Kustomize
+```shell
+curl -s "https://raw.githubusercontent.com/\
+kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+ 
+mv kustomize /usr/loca/bin/
```
Comment (jzding) [inline docs/development.md]: typo `local`

File: docs/development.md [hunk 3]
```diff
@@ -32,7 +32,20 @@ Use producer.yaml,consumer.yaml and service.yaml from examples/manifests folder
...
 
 
+### Deploying examples using kustomize
 
+Install Kustomize
+``` 
+$ curl -s "https://raw.githubusercontent.com/\
+kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+ 
+$mv kustomize /usr/loca/bin/
```
[the remark this answers is not in the document]
Comment (aneeshkp) [reply docs/development.md]: Fixed

File: docs/development.md (hunk 1 above)
Comment (aneeshkp) [reply docs/development.md]: changed

**Pull request merged**